proposals/src/passive-stake-delegation-and-rewards.md

```diff
@@ -17,8 +17,8 @@ validator.
 
 In the current design each staker creates their own VoteState, and assigns a
 **delegate** in the VoteState that can submit votes.  Since the validator has to
-actively vote for each stake delegated to it, it can censor stakes by not voting
-for them.
+actively vote for each stake delegated to it, validators can censor stakes by
+not voting to them.
```

Comment:
voting to them?

Reply:
"to" because a validator has to issue a transaction to every instance of the VoteProgram that delegates to the validator

Reply:
for them?